pkg/localize/locales/en/cmd/kafka_create.en.toml

```diff
@@ -69,6 +69,9 @@ one = "Geographical region where the Kafka instance will be deployed"
 [kafka.create.argument.name.error.requiredWhenNonInteractive]
 one = 'name is required. Run "rhoas kafka create <name>"'
 
+[kafka.create.flags.notAllowedWhenInteractive]
+one = 'region and provider flags not allowed for interactive Kafka instance creation'
```

Comment:
Suggestion for a better error message @craicoverflow ?

Reply:
Swap it round like something like this: `"name" argument is required when creating a Kafka instance`

Reply:
Changed it to `'name is required. Run "rhoas kafka create <name>"'`